sdk/compute/Azure.ResourceManager.Compute/tests/Scenario/VirtualMachineManagedIdentityTests.cs

```diff
@@ -95,6 +95,7 @@ public async Task UpdateVmIdentityFromNoneToSystem()
             var vmName = Recording.GenerateAssetName("testVM-");
             var nic = await CreateBasicDependenciesOfVirtualMachineAsync();
             var input = ResourceDataHelper.GetBasicLinuxVirtualMachineData(DefaultLocation, vmName, nic.Id);
+            input.Tags.Add("skip-cloudgov-VM_SI_Deploy_GuestConfig_Extension", "true");
```

Comment:
I think we would better to add a comment here describing this magic tag, otherwise in the future people would get confused by this.

Reply:
done